1.9.0

```diff
@@ -258,6 +258,40 @@
           }
         }
       ]
+    },
+    {
+      "sobjectName":"Campaign",
+      "instructionTypeName":"mdTaskListView",
+      "taskList":[
+        {
+          "jiraTicket": "AC-59235",
+          "description": "Delete AllActiveCampaigns List View",
+          "actionType": "deleteListView",
+          "optionsMap": {
+            "fullName": "Campaign.AllActiveCampaigns",
```

Comment:
Use the actionTypeName parameter instead of fullName. actionTypeName is used to hold the name of whatever we're trying to operate on.